ansible/roles/kstest/tasks/main.yml

```diff
@@ -24,6 +24,11 @@
       - rsync
     state: latest
 
+- name: Update all packages
+  dnf:
+    name: "*"
+    state: latest
```

Comment:
When is this called?
Is it called only when a host is created or every time we are launching a test? I guess the first one is correct but want to be sure.

Reply:
Yes, the first one is true.